[2023-11-26T00:17:32.463+0000] {processor.py:157} INFO - Started process (PID=12842) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:17:32.470+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T00:17:32.493+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:17:32.492+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:17:33.685+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:17:33.730+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:17:33.729+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T00:17:33.758+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:17:33.758+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T00:17:33.771+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 1.333 seconds
[2023-11-26T00:50:45.751+0000] {processor.py:157} INFO - Started process (PID=12856) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:50:45.755+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T00:50:45.789+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:50:45.782+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:50:47.153+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:50:47.222+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:50:47.221+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T00:50:47.239+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:50:47.239+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T00:50:47.251+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 1.532 seconds
[2023-11-26T00:51:03.761+0000] {processor.py:157} INFO - Started process (PID=12866) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:51:03.763+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T00:51:03.770+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:51:03.769+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:51:04.232+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:51:04.271+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:51:04.270+0000] {taskinstance.py:1939} ERROR - {'DAG Id': 'outcomes_dag', 'Task Id': 'load_animal_dim', 'Run Id': 'scheduled__2023-11-25T00:00:00+00:00', 'Hostname': '1df7871f6344', 'External Executor Id': '68ab3229-7510-4d5e-87e2-4ad053988e95'}
[2023-11-26T00:51:04.281+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:51:04.281+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=outcomes_dag, task_id=load_animal_dim, execution_date=20231125T000000, start_date=20231126T001754, end_date=20231126T005104
[2023-11-26T00:51:04.288+0000] {processor.py:790} INFO - Executed failure callback for <TaskInstance: outcomes_dag.load_animal_dim scheduled__2023-11-25T00:00:00+00:00 [failed]> in state failed
[2023-11-26T00:51:04.302+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:51:04.302+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T00:51:04.313+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:51:04.313+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T00:51:04.320+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.566 seconds
[2023-11-26T00:55:52.277+0000] {processor.py:157} INFO - Started process (PID=12868) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:55:52.278+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T00:55:52.281+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:55:52.280+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:55:52.529+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T00:55:52.550+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:55:52.550+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T00:55:52.564+0000] {logging_mixin.py:154} INFO - [2023-11-26T00:55:52.564+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T00:55:52.574+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.301 seconds
[2023-11-26T01:06:32.333+0000] {processor.py:157} INFO - Started process (PID=12877) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:06:32.340+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:06:32.376+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:06:32.372+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:06:33.752+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:06:33.836+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:06:33.834+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:06:33.888+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:06:33.887+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:06:33.921+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 1.597 seconds
[2023-11-26T01:17:29.989+0000] {processor.py:157} INFO - Started process (PID=12886) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:17:29.990+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:17:29.994+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:17:29.994+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:17:31.333+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:17:31.453+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:17:31.448+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:17:31.549+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:17:31.549+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:17:31.592+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 1.609 seconds
[2023-11-26T01:35:14.333+0000] {processor.py:157} INFO - Started process (PID=12895) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:35:14.336+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:35:14.356+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:35:14.355+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:35:15.025+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:35:15.101+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:35:15.101+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:35:15.170+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:35:15.170+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:35:15.188+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.862 seconds
[2023-11-26T01:35:45.745+0000] {processor.py:157} INFO - Started process (PID=12904) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:35:45.746+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:35:45.751+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:35:45.750+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:35:46.011+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:35:46.051+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:35:46.050+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:35:46.068+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:35:46.068+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:35:46.080+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.343 seconds
[2023-11-26T01:40:23.468+0000] {processor.py:157} INFO - Started process (PID=12913) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:40:23.470+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:40:23.478+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:40:23.476+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:40:24.122+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:40:24.198+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:40:24.198+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:40:24.230+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:40:24.230+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:40:24.244+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.782 seconds
[2023-11-26T01:50:46.951+0000] {processor.py:157} INFO - Started process (PID=12922) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:50:46.952+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:50:46.956+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:50:46.956+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:50:47.402+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:50:47.509+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:50:47.508+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:50:47.665+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:50:47.665+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:50:47.743+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.797 seconds
[2023-11-26T01:51:19.384+0000] {processor.py:157} INFO - Started process (PID=12931) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:51:19.387+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:51:19.402+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:51:19.401+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:51:19.845+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:51:19.888+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:51:19.888+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:51:19.937+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:51:19.936+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:51:19.961+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.584 seconds
[2023-11-26T01:51:50.359+0000] {processor.py:157} INFO - Started process (PID=12940) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:51:50.360+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:51:50.364+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:51:50.363+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:51:50.568+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:51:50.588+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:51:50.588+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:51:50.602+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:51:50.602+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:51:50.610+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.256 seconds
[2023-11-26T01:52:20.912+0000] {processor.py:157} INFO - Started process (PID=12949) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:52:20.915+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:52:20.921+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:52:20.920+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:52:21.294+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:52:21.326+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:52:21.326+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:52:21.344+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:52:21.344+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:52:21.354+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.466 seconds
[2023-11-26T01:52:51.771+0000] {processor.py:157} INFO - Started process (PID=12958) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:52:51.772+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:52:51.776+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:52:51.776+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:52:52.096+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:52:52.125+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:52:52.125+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:52:52.144+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:52:52.143+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:52:52.156+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.391 seconds
[2023-11-26T01:53:22.604+0000] {processor.py:157} INFO - Started process (PID=12972) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:53:22.607+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:53:22.616+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:53:22.612+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:53:24.941+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:53:24.987+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:53:24.986+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:53:25.007+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:53:25.007+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:53:25.021+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 2.426 seconds
[2023-11-26T01:53:29.496+0000] {processor.py:157} INFO - Started process (PID=12974) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:53:29.501+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:53:29.505+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:53:29.505+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:53:29.760+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:53:29.812+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:53:29.811+0000] {taskinstance.py:1939} ERROR - {'DAG Id': 'outcomes_dag', 'Task Id': 'load_animal_dim', 'Run Id': 'manual__2023-11-26T01:52:53.510743+00:00', 'Hostname': '1df7871f6344', 'External Executor Id': '3f8a0032-6130-416b-88b4-f7e6f50b109d'}
[2023-11-26T01:53:29.822+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:53:29.821+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=outcomes_dag, task_id=load_animal_dim, execution_date=20231126T015253, start_date=20231126T015311, end_date=20231126T015329
[2023-11-26T01:53:29.828+0000] {processor.py:790} INFO - Executed failure callback for <TaskInstance: outcomes_dag.load_animal_dim manual__2023-11-26T01:52:53.510743+00:00 [failed]> in state failed
[2023-11-26T01:53:29.840+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:53:29.840+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:53:29.850+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:53:29.850+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:53:29.863+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.393 seconds
[2023-11-26T01:54:37.724+0000] {processor.py:157} INFO - Started process (PID=34) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:54:37.726+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:54:37.731+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:54:37.731+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:54:38.595+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:54:38.829+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:54:38.829+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:54:38.893+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:54:38.893+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:54:38.930+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 1.217 seconds
[2023-11-26T01:55:09.284+0000] {processor.py:157} INFO - Started process (PID=43) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:55:09.286+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:55:09.301+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:55:09.298+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:55:09.963+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:55:09.979+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:55:09.979+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:55:09.990+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:55:09.990+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:55:09.998+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.721 seconds
[2023-11-26T01:55:40.337+0000] {processor.py:157} INFO - Started process (PID=52) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:55:40.338+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:55:40.342+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:55:40.341+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:55:40.811+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:55:40.837+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:55:40.836+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:55:40.849+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:55:40.848+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:55:40.856+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.525 seconds
[2023-11-26T01:56:10.965+0000] {processor.py:157} INFO - Started process (PID=61) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:56:10.967+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:56:10.973+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:56:10.972+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:56:11.415+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:56:11.431+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:56:11.431+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:56:11.442+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:56:11.442+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:56:11.450+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.491 seconds
[2023-11-26T01:56:41.836+0000] {processor.py:157} INFO - Started process (PID=70) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:56:41.838+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:56:41.843+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:56:41.842+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:56:42.280+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:56:42.311+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:56:42.310+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:56:42.332+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:56:42.332+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:56:42.342+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.514 seconds
[2023-11-26T01:57:12.766+0000] {processor.py:157} INFO - Started process (PID=79) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:57:12.769+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:57:12.775+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:57:12.775+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:57:13.180+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:57:13.199+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:57:13.199+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:57:13.212+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:57:13.212+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:57:13.221+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.464 seconds
[2023-11-26T01:57:43.589+0000] {processor.py:157} INFO - Started process (PID=88) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:57:43.591+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:57:43.595+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:57:43.595+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:57:44.016+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:57:44.046+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:57:44.045+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:57:44.060+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:57:44.060+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:57:44.068+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.485 seconds
[2023-11-26T01:58:14.499+0000] {processor.py:157} INFO - Started process (PID=97) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:58:14.502+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:58:14.509+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:58:14.508+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:58:14.965+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:58:14.986+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:58:14.986+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:58:15.005+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:58:15.005+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:58:15.015+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.524 seconds
[2023-11-26T01:58:45.423+0000] {processor.py:157} INFO - Started process (PID=106) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:58:45.425+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:58:45.431+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:58:45.430+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:58:45.827+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:58:45.849+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:58:45.849+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:58:45.861+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:58:45.861+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:58:45.869+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.452 seconds
[2023-11-26T01:59:16.214+0000] {processor.py:157} INFO - Started process (PID=115) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:59:16.217+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:59:16.222+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:59:16.222+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:59:16.566+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:59:16.581+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:59:16.581+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:59:16.602+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:59:16.601+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:59:16.617+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.412 seconds
[2023-11-26T01:59:46.979+0000] {processor.py:157} INFO - Started process (PID=124) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:59:46.981+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T01:59:46.996+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:59:46.994+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:59:47.417+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T01:59:47.437+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:59:47.437+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T01:59:47.448+0000] {logging_mixin.py:154} INFO - [2023-11-26T01:59:47.447+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T01:59:47.456+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.481 seconds
[2023-11-26T02:00:17.849+0000] {processor.py:157} INFO - Started process (PID=133) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:00:17.853+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:00:17.858+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:00:17.857+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:00:18.258+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:00:18.274+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:00:18.274+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:00:18.286+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:00:18.286+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:00:18.300+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.458 seconds
[2023-11-26T02:00:48.666+0000] {processor.py:157} INFO - Started process (PID=142) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:00:48.668+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:00:48.671+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:00:48.671+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:00:49.136+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:00:49.154+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:00:49.154+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:00:49.167+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:00:49.167+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:00:49.176+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.515 seconds
[2023-11-26T02:01:19.548+0000] {processor.py:157} INFO - Started process (PID=151) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:01:19.550+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:01:19.552+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:01:19.552+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:01:20.023+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:01:20.037+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:01:20.037+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:01:20.049+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:01:20.049+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:01:20.056+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.511 seconds
[2023-11-26T02:01:50.398+0000] {processor.py:157} INFO - Started process (PID=160) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:01:50.405+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:01:50.420+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:01:50.419+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:01:51.011+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:01:51.031+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:01:51.031+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:01:51.045+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:01:51.045+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:01:51.069+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.681 seconds
[2023-11-26T02:02:21.480+0000] {processor.py:157} INFO - Started process (PID=169) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:02:21.482+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:02:21.485+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:02:21.485+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:02:21.947+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:02:21.998+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:02:21.998+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:02:22.019+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:02:22.019+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:02:22.036+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.569 seconds
[2023-11-26T02:02:52.093+0000] {processor.py:157} INFO - Started process (PID=178) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:02:52.094+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:02:52.097+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:02:52.096+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:02:52.570+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:02:52.597+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:02:52.597+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:02:52.611+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:02:52.611+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:02:52.622+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.533 seconds
[2023-11-26T02:03:22.801+0000] {processor.py:157} INFO - Started process (PID=187) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:03:22.804+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:03:22.808+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:03:22.807+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:03:23.204+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:03:23.225+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:03:23.225+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:03:23.238+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:03:23.237+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:03:23.247+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.453 seconds
[2023-11-26T02:03:53.343+0000] {processor.py:157} INFO - Started process (PID=196) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:03:53.346+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:03:53.350+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:03:53.350+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:03:53.756+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:03:53.777+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:03:53.777+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:03:53.790+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:03:53.790+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:03:53.799+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.465 seconds
[2023-11-26T02:04:23.969+0000] {processor.py:157} INFO - Started process (PID=205) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:04:23.970+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:04:23.974+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:04:23.974+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:04:24.397+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:04:24.417+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:04:24.417+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:04:24.431+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:04:24.430+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:04:24.439+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.480 seconds
[2023-11-26T02:04:54.847+0000] {processor.py:157} INFO - Started process (PID=214) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:04:54.850+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:04:54.855+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:04:54.854+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:04:55.253+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:04:55.271+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:04:55.271+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:04:55.282+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:04:55.282+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:04:55.290+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.452 seconds
[2023-11-26T02:05:25.637+0000] {processor.py:157} INFO - Started process (PID=223) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:05:25.640+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:05:25.646+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:05:25.645+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:05:26.071+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:05:26.089+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:05:26.089+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:05:26.101+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:05:26.101+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:05:26.110+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.480 seconds
[2023-11-26T02:05:56.503+0000] {processor.py:157} INFO - Started process (PID=232) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:05:56.506+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:05:56.512+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:05:56.511+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:05:56.922+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:05:56.941+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:05:56.941+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:05:56.952+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:05:56.952+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:05:56.961+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.465 seconds
[2023-11-26T02:06:27.313+0000] {processor.py:157} INFO - Started process (PID=241) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:06:27.314+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:06:27.318+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:06:27.317+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:06:27.725+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:06:27.752+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:06:27.752+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:06:27.773+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:06:27.773+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:06:27.784+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.477 seconds
[2023-11-26T02:06:58.223+0000] {processor.py:157} INFO - Started process (PID=250) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:06:58.226+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:06:58.232+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:06:58.231+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:06:58.638+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:06:58.654+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:06:58.654+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:06:58.668+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:06:58.668+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:06:58.677+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.462 seconds
[2023-11-26T02:07:29.042+0000] {processor.py:157} INFO - Started process (PID=259) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:07:29.043+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:07:29.047+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:07:29.047+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:07:29.522+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:07:29.543+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:07:29.543+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:07:29.559+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:07:29.559+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:07:29.569+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.530 seconds
[2023-11-26T02:07:59.989+0000] {processor.py:157} INFO - Started process (PID=268) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:07:59.992+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:07:59.997+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:07:59.996+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:08:00.388+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:08:00.405+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:08:00.405+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:08:00.417+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:08:00.417+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:08:00.425+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.442 seconds
[2023-11-26T02:08:30.673+0000] {processor.py:157} INFO - Started process (PID=277) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:08:30.675+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:08:30.679+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:08:30.679+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:08:31.073+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:08:31.092+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:08:31.092+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:08:31.103+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:08:31.103+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:08:31.112+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.445 seconds
[2023-11-26T02:09:01.537+0000] {processor.py:157} INFO - Started process (PID=286) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:09:01.540+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:09:01.544+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:09:01.544+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:09:01.929+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:09:01.951+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:09:01.951+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:09:01.964+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:09:01.964+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:09:01.972+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.443 seconds
[2023-11-26T02:09:32.318+0000] {processor.py:157} INFO - Started process (PID=295) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:09:32.321+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:09:32.326+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:09:32.325+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:09:32.752+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:09:32.769+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:09:32.769+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:09:32.781+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:09:32.781+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:09:32.791+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.479 seconds
[2023-11-26T02:10:03.188+0000] {processor.py:157} INFO - Started process (PID=311) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:10:03.190+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:10:03.195+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:10:03.194+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:10:03.562+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:10:03.581+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:10:03.581+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:10:03.595+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:10:03.595+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:10:03.603+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.423 seconds
[2023-11-26T02:10:33.978+0000] {processor.py:157} INFO - Started process (PID=320) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:10:33.981+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:10:33.986+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:10:33.985+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:10:34.370+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:10:34.387+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:10:34.387+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:10:34.399+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:10:34.399+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:10:34.406+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.436 seconds
[2023-11-26T02:11:04.738+0000] {processor.py:157} INFO - Started process (PID=329) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:11:04.742+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:11:04.747+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:11:04.746+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:11:05.185+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:11:05.206+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:11:05.206+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:11:05.218+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:11:05.218+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:11:05.227+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.497 seconds
[2023-11-26T02:11:35.613+0000] {processor.py:157} INFO - Started process (PID=338) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:11:35.615+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:11:35.621+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:11:35.620+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:11:36.028+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:11:36.056+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:11:36.056+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:11:36.076+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:11:36.075+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:11:36.083+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.480 seconds
[2023-11-26T02:12:06.460+0000] {processor.py:157} INFO - Started process (PID=347) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:12:06.462+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:12:06.466+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:12:06.466+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:12:06.877+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:12:06.893+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:12:06.893+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:12:06.905+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:12:06.905+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:12:06.913+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.459 seconds
[2023-11-26T02:12:37.248+0000] {processor.py:157} INFO - Started process (PID=356) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:12:37.251+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:12:37.255+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:12:37.255+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:12:37.669+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:12:37.689+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:12:37.689+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:12:37.700+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:12:37.699+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:12:37.708+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.466 seconds
[2023-11-26T02:13:08.041+0000] {processor.py:157} INFO - Started process (PID=365) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:13:08.043+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:13:08.046+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:13:08.046+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:13:08.449+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:13:08.488+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:13:08.488+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:13:08.502+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:13:08.502+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:13:08.510+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.474 seconds
[2023-11-26T02:13:38.866+0000] {processor.py:157} INFO - Started process (PID=374) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:13:38.871+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:13:38.878+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:13:38.877+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:13:39.309+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:13:39.328+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:13:39.328+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:13:39.340+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:13:39.340+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:13:39.347+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.493 seconds
[2023-11-26T02:14:09.724+0000] {processor.py:157} INFO - Started process (PID=384) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:14:09.725+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:14:09.728+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:14:09.728+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:14:10.163+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:14:10.184+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:14:10.184+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:14:10.197+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:14:10.197+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:14:10.209+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.503 seconds
[2023-11-26T02:14:40.770+0000] {processor.py:157} INFO - Started process (PID=393) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:14:40.772+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:14:40.784+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:14:40.783+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:14:41.293+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:14:41.312+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:14:41.312+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:14:41.323+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:14:41.323+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:14:41.331+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.566 seconds
[2023-11-26T02:15:11.728+0000] {processor.py:157} INFO - Started process (PID=402) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:15:11.729+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:15:11.734+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:15:11.733+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:15:12.149+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:15:12.166+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:15:12.166+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:15:12.177+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:15:12.177+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:15:12.186+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.465 seconds
[2023-11-26T02:15:42.552+0000] {processor.py:157} INFO - Started process (PID=411) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:15:42.555+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:15:42.566+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:15:42.562+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:15:42.969+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:15:42.992+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:15:42.992+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:15:43.011+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:15:43.011+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:15:43.019+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.471 seconds
[2023-11-26T02:16:13.446+0000] {processor.py:157} INFO - Started process (PID=420) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:16:13.449+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:16:13.456+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:16:13.455+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:16:13.900+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:16:13.917+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:16:13.917+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:16:13.930+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:16:13.929+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:16:13.937+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.530 seconds
[2023-11-26T02:16:44.304+0000] {processor.py:157} INFO - Started process (PID=428) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:16:44.305+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:16:44.307+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:16:44.307+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:16:44.698+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:16:44.718+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:16:44.718+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:16:44.729+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:16:44.729+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:16:44.737+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.437 seconds
[2023-11-26T02:17:15.146+0000] {processor.py:157} INFO - Started process (PID=437) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:15.148+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:17:15.152+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:17:15.152+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:15.567+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:15.584+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:17:15.584+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:17:15.594+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:17:15.594+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:17:15.602+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.462 seconds
[2023-11-26T02:17:45.956+0000] {processor.py:157} INFO - Started process (PID=446) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:45.958+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:17:45.963+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:17:45.963+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:46.398+0000] {processor.py:839} INFO - DAG(s) dict_keys(['outcomes_dag']) retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:46.426+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:17:46.426+0000] {dag.py:2941} INFO - Sync 1 DAGs
[2023-11-26T02:17:46.439+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:17:46.439+0000] {dag.py:3722} INFO - Setting next_dagrun for outcomes_dag to 2023-11-26T00:00:00+00:00, run_after=2023-11-27T00:00:00+00:00
[2023-11-26T02:17:46.451+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.502 seconds
[2023-11-26T02:17:50.033+0000] {processor.py:157} INFO - Started process (PID=448) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:50.035+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:17:50.038+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:17:50.038+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:50.385+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:17:50.382+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:17:50.385+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:17:50.399+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.373 seconds
[2023-11-26T02:18:20.767+0000] {processor.py:157} INFO - Started process (PID=456) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:18:20.770+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:18:20.774+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:18:20.773+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:18:21.253+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:18:21.252+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:18:21.253+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:18:21.270+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.513 seconds
[2023-11-26T02:18:51.667+0000] {processor.py:157} INFO - Started process (PID=465) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:18:51.671+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:18:51.675+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:18:51.674+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:18:52.100+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:18:52.099+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:18:52.101+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:18:52.119+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.473 seconds
[2023-11-26T02:19:22.490+0000] {processor.py:157} INFO - Started process (PID=474) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:19:22.492+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:19:22.495+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:19:22.494+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:19:22.888+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:19:22.888+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:19:22.889+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:19:22.904+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.418 seconds
[2023-11-26T02:19:53.277+0000] {processor.py:157} INFO - Started process (PID=483) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:19:53.279+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:19:53.280+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:19:53.280+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:19:53.673+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:19:53.672+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:19:53.674+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:19:53.686+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.415 seconds
[2023-11-26T02:20:24.052+0000] {processor.py:157} INFO - Started process (PID=492) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:20:24.055+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:20:24.057+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:20:24.056+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:20:24.433+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:20:24.432+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:20:24.433+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:20:24.445+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.400 seconds
[2023-11-26T02:20:54.849+0000] {processor.py:157} INFO - Started process (PID=501) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:20:54.852+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:20:54.853+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:20:54.853+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:20:55.264+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:20:55.263+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:20:55.264+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:20:55.283+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.441 seconds
[2023-11-26T02:21:25.643+0000] {processor.py:157} INFO - Started process (PID=510) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:21:25.645+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:21:25.646+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:21:25.646+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:21:26.057+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:21:26.056+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:21:26.057+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:21:26.071+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.437 seconds
[2023-11-26T02:21:56.495+0000] {processor.py:157} INFO - Started process (PID=519) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:21:56.498+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:21:56.499+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:21:56.499+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:21:56.939+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:21:56.939+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:21:56.940+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:21:56.956+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.469 seconds
[2023-11-26T02:22:27.353+0000] {processor.py:157} INFO - Started process (PID=528) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:22:27.355+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:22:27.357+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:22:27.356+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:22:27.792+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:22:27.790+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:22:27.792+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:22:27.808+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.459 seconds
[2023-11-26T02:22:31.427+0000] {processor.py:157} INFO - Started process (PID=530) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:22:31.429+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:22:31.430+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:22:31.430+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:22:31.809+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:22:31.808+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:22:31.809+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:22:31.822+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.401 seconds
[2023-11-26T02:23:02.324+0000] {processor.py:157} INFO - Started process (PID=539) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:23:02.330+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:23:02.332+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:23:02.331+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:23:02.994+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:23:02.993+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:23:02.995+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:23:03.011+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.776 seconds
[2023-11-26T02:23:33.278+0000] {processor.py:157} INFO - Started process (PID=548) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:23:33.282+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:23:33.283+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:23:33.283+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:23:33.658+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:23:33.655+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:23:33.658+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:23:33.677+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.405 seconds
[2023-11-26T02:24:03.790+0000] {processor.py:157} INFO - Started process (PID=557) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:24:03.792+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:24:03.794+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:24:03.793+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:24:04.208+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:24:04.207+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:24:04.208+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:24:04.234+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.451 seconds
[2023-11-26T02:24:34.351+0000] {processor.py:157} INFO - Started process (PID=566) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:24:34.352+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:24:34.353+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:24:34.353+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:24:34.790+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:24:34.789+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:24:34.790+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:24:34.802+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.459 seconds
[2023-11-26T02:25:05.000+0000] {processor.py:157} INFO - Started process (PID=575) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:25:05.002+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:25:05.003+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:25:05.002+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:25:05.425+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:25:05.424+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:25:05.425+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:25:05.436+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.443 seconds
[2023-11-26T02:25:35.613+0000] {processor.py:157} INFO - Started process (PID=584) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:25:35.615+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:25:35.617+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:25:35.616+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:25:36.050+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:25:36.049+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:25:36.050+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:25:36.062+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.457 seconds
[2023-11-26T02:26:06.202+0000] {processor.py:157} INFO - Started process (PID=599) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:26:06.204+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:26:06.208+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:26:06.206+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:26:06.614+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:26:06.613+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:26:06.615+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:26:06.629+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.432 seconds
[2023-11-26T02:26:36.733+0000] {processor.py:157} INFO - Started process (PID=608) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:26:36.735+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:26:36.736+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:26:36.735+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:26:37.127+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:26:37.126+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:26:37.127+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:26:37.138+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.419 seconds
[2023-11-26T02:27:07.431+0000] {processor.py:157} INFO - Started process (PID=617) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:27:07.433+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:27:07.434+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:27:07.434+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:27:07.793+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:27:07.792+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:27:07.794+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:27:07.805+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.380 seconds
[2023-11-26T02:27:38.151+0000] {processor.py:157} INFO - Started process (PID=626) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:27:38.153+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:27:38.155+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:27:38.154+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:27:38.526+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:27:38.525+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:27:38.526+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:27:38.537+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.393 seconds
[2023-11-26T02:28:08.951+0000] {processor.py:157} INFO - Started process (PID=635) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:28:08.952+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:28:08.954+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:28:08.953+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:28:09.329+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:28:09.328+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:28:09.330+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:28:09.341+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.396 seconds
[2023-11-26T02:28:39.699+0000] {processor.py:157} INFO - Started process (PID=644) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:28:39.701+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:28:39.702+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:28:39.702+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:28:40.086+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:28:40.085+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:28:40.086+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:28:40.097+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.405 seconds
[2023-11-26T02:29:10.481+0000] {processor.py:157} INFO - Started process (PID=653) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:29:10.483+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:29:10.484+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:29:10.483+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:29:10.883+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:29:10.882+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:29:10.883+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:29:10.897+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.425 seconds
[2023-11-26T02:29:41.256+0000] {processor.py:157} INFO - Started process (PID=662) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:29:41.259+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:29:41.262+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:29:41.260+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:29:41.677+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:29:41.676+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:29:41.678+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:29:41.705+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.456 seconds
[2023-11-26T02:30:12.102+0000] {processor.py:157} INFO - Started process (PID=671) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:30:12.104+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:30:12.106+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:30:12.105+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:30:12.528+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:30:12.527+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:30:12.528+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:30:12.540+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.456 seconds
[2023-11-26T02:30:42.841+0000] {processor.py:157} INFO - Started process (PID=680) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:30:42.844+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:30:42.846+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:30:42.845+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:30:43.245+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:30:43.244+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:30:43.245+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:30:43.259+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.425 seconds
[2023-11-26T02:31:13.640+0000] {processor.py:157} INFO - Started process (PID=689) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:31:13.641+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:31:13.642+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:31:13.642+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:31:14.039+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:31:14.038+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:31:14.039+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:31:14.054+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.422 seconds
[2023-11-26T02:31:44.474+0000] {processor.py:157} INFO - Started process (PID=698) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:31:44.480+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:31:44.481+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:31:44.480+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:31:44.896+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:31:44.895+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:31:44.896+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:31:44.908+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.457 seconds
[2023-11-26T02:32:15.224+0000] {processor.py:157} INFO - Started process (PID=707) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:32:15.225+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:32:15.226+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:32:15.226+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:32:15.639+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:32:15.637+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:32:15.640+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:32:15.653+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.434 seconds
[2023-11-26T02:32:46.255+0000] {processor.py:157} INFO - Started process (PID=716) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:32:46.260+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:32:46.262+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:32:46.261+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:32:46.899+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:32:46.898+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:32:46.900+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:32:46.924+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.693 seconds
[2023-11-26T02:33:17.300+0000] {processor.py:157} INFO - Started process (PID=725) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:33:17.302+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:33:17.303+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:33:17.303+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:33:17.685+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:33:17.684+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:33:17.685+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:33:17.699+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.406 seconds
[2023-11-26T02:33:48.076+0000] {processor.py:157} INFO - Started process (PID=734) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:33:48.081+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:33:48.083+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:33:48.082+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:33:48.462+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:33:48.461+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:33:48.462+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:33:48.472+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.403 seconds
[2023-11-26T02:34:18.876+0000] {processor.py:157} INFO - Started process (PID=743) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:34:18.878+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:34:18.880+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:34:18.879+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:34:19.233+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:34:19.233+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:34:19.234+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:34:19.245+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.376 seconds
[2023-11-26T02:34:49.677+0000] {processor.py:157} INFO - Started process (PID=752) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:34:49.688+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:34:49.693+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:34:49.691+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:34:50.085+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:34:50.084+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:34:50.085+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:34:50.099+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.441 seconds
[2023-11-26T02:35:20.414+0000] {processor.py:157} INFO - Started process (PID=761) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:35:20.416+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:35:20.418+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:35:20.417+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:35:20.795+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:35:20.794+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:35:20.795+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:35:20.807+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.398 seconds
[2023-11-26T02:35:51.149+0000] {processor.py:157} INFO - Started process (PID=770) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:35:51.151+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:35:51.152+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:35:51.152+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:35:51.487+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:35:51.486+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:35:51.487+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:35:51.500+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.355 seconds
[2023-11-26T02:36:21.842+0000] {processor.py:157} INFO - Started process (PID=780) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:36:21.845+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:36:21.846+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:36:21.845+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:36:22.199+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:36:22.198+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:36:22.199+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:36:22.211+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.377 seconds
[2023-11-26T02:36:52.631+0000] {processor.py:157} INFO - Started process (PID=789) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:36:52.634+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:36:52.635+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:36:52.634+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:36:53.069+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:36:53.068+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:36:53.070+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:36:53.081+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.465 seconds
[2023-11-26T02:37:23.442+0000] {processor.py:157} INFO - Started process (PID=798) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:37:23.443+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:37:23.444+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:37:23.444+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:37:23.994+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:37:23.993+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:37:23.995+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:37:24.029+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.591 seconds
[2023-11-26T02:37:54.455+0000] {processor.py:157} INFO - Started process (PID=806) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:37:54.456+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:37:54.457+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:37:54.457+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:37:54.891+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:37:54.890+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:37:54.891+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:37:54.905+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.453 seconds
[2023-11-26T02:38:25.238+0000] {processor.py:157} INFO - Started process (PID=815) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:38:25.239+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:38:25.240+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:38:25.240+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:38:25.657+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:38:25.656+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:38:25.657+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:38:25.683+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.449 seconds
[2023-11-26T02:38:56.046+0000] {processor.py:157} INFO - Started process (PID=824) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:38:56.048+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:38:56.049+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:38:56.049+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:38:56.481+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:38:56.480+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:38:56.482+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:38:56.497+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.454 seconds
[2023-11-26T02:39:26.928+0000] {processor.py:157} INFO - Started process (PID=833) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:39:26.930+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:39:26.931+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:39:26.930+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:39:27.369+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:39:27.368+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:39:27.370+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:39:27.385+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.465 seconds
[2023-11-26T02:39:57.889+0000] {processor.py:157} INFO - Started process (PID=842) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:39:57.892+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:39:57.893+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:39:57.893+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:39:58.320+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:39:58.319+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:39:58.320+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:39:58.332+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.449 seconds
[2023-11-26T02:40:28.697+0000] {processor.py:157} INFO - Started process (PID=851) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:40:28.698+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:40:28.700+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:40:28.699+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:40:29.110+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:40:29.109+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:40:29.111+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:40:29.128+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.437 seconds
[2023-11-26T02:40:59.535+0000] {processor.py:157} INFO - Started process (PID=860) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:40:59.537+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:40:59.539+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:40:59.538+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:40:59.943+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:40:59.942+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:40:59.943+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:40:59.972+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.446 seconds
[2023-11-26T02:41:30.317+0000] {processor.py:157} INFO - Started process (PID=869) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:41:30.319+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:41:30.320+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:41:30.320+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:41:30.702+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:41:30.701+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:41:30.702+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:41:30.714+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.400 seconds
[2023-11-26T02:42:01.082+0000] {processor.py:157} INFO - Started process (PID=878) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:42:01.085+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:42:01.086+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:42:01.085+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:42:01.520+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:42:01.518+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:42:01.520+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:42:01.532+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.456 seconds
[2023-11-26T02:42:31.912+0000] {processor.py:157} INFO - Started process (PID=887) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:42:31.913+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:42:31.915+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:42:31.914+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:42:32.318+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:42:32.317+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:42:32.319+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:42:32.342+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.436 seconds
[2023-11-26T02:43:02.742+0000] {processor.py:157} INFO - Started process (PID=896) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:43:02.747+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:43:02.749+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:43:02.748+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:43:03.149+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:43:03.148+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:43:03.149+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:43:03.160+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.425 seconds
[2023-11-26T02:43:33.497+0000] {processor.py:157} INFO - Started process (PID=905) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:43:33.500+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:43:33.502+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:43:33.501+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:43:33.902+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:43:33.901+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:43:33.903+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:43:33.916+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.426 seconds
[2023-11-26T02:44:04.344+0000] {processor.py:157} INFO - Started process (PID=914) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:44:04.346+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:44:04.347+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:44:04.347+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:44:04.734+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:44:04.734+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:44:04.735+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:44:04.747+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.409 seconds
[2023-11-26T02:44:35.079+0000] {processor.py:157} INFO - Started process (PID=923) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:44:35.081+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:44:35.083+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:44:35.082+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:44:35.468+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:44:35.467+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:44:35.469+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:44:35.483+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.411 seconds
[2023-11-26T02:45:05.834+0000] {processor.py:157} INFO - Started process (PID=932) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:45:05.837+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:45:05.839+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:45:05.838+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:45:06.222+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:45:06.221+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:45:06.222+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:45:06.236+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.407 seconds
[2023-11-26T02:45:36.634+0000] {processor.py:157} INFO - Started process (PID=941) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:45:36.637+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:45:36.638+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:45:36.638+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:45:37.053+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:45:37.053+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:45:37.054+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:45:37.067+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.439 seconds
[2023-11-26T02:46:07.456+0000] {processor.py:157} INFO - Started process (PID=950) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:46:07.458+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:46:07.460+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:46:07.459+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:46:07.868+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:46:07.868+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:46:07.869+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:46:07.883+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.434 seconds
[2023-11-26T02:46:38.311+0000] {processor.py:157} INFO - Started process (PID=959) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:46:38.313+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:46:38.315+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:46:38.314+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:46:38.746+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:46:38.745+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:46:38.746+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:46:38.760+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.456 seconds
[2023-11-26T02:47:09.155+0000] {processor.py:157} INFO - Started process (PID=968) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:47:09.158+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:47:09.159+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:47:09.159+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:47:09.550+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:47:09.549+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:47:09.550+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:47:09.562+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.413 seconds
[2023-11-26T02:47:39.952+0000] {processor.py:157} INFO - Started process (PID=977) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:47:39.954+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:47:39.955+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:47:39.955+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:47:40.379+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:47:40.377+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:47:40.379+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:47:40.402+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.456 seconds
[2023-11-26T02:48:10.725+0000] {processor.py:157} INFO - Started process (PID=993) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:48:10.727+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:48:10.728+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:48:10.727+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:48:11.102+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:48:11.101+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:48:11.103+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:48:11.115+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.393 seconds
[2023-11-26T02:48:41.500+0000] {processor.py:157} INFO - Started process (PID=1002) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:48:41.501+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:48:41.503+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:48:41.502+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:48:41.918+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:48:41.916+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:48:41.918+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:48:41.934+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.439 seconds
[2023-11-26T02:49:12.253+0000] {processor.py:157} INFO - Started process (PID=1011) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:49:12.255+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:49:12.257+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:49:12.256+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:49:12.678+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:49:12.677+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:49:12.679+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:49:12.692+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.445 seconds
[2023-11-26T02:49:43.101+0000] {processor.py:157} INFO - Started process (PID=1020) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:49:43.103+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:49:43.104+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:49:43.104+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:49:43.524+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:49:43.523+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:49:43.524+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:49:43.538+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.444 seconds
[2023-11-26T02:50:13.864+0000] {processor.py:157} INFO - Started process (PID=1029) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:50:13.866+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:50:13.867+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:50:13.867+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:50:14.417+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:50:14.415+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:50:14.417+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:50:14.448+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.590 seconds
[2023-11-26T02:50:44.882+0000] {processor.py:157} INFO - Started process (PID=1038) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:50:44.885+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:50:44.887+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:50:44.886+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:50:45.305+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:50:45.304+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:50:45.305+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:50:45.318+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.444 seconds
[2023-11-26T02:51:15.774+0000] {processor.py:157} INFO - Started process (PID=1047) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:51:15.775+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:51:15.777+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:51:15.776+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:51:16.166+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:51:16.165+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:51:16.166+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:51:16.179+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.412 seconds
[2023-11-26T02:51:46.618+0000] {processor.py:157} INFO - Started process (PID=1056) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:51:46.620+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:51:46.621+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:51:46.621+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:51:47.082+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:51:47.081+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:51:47.082+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:51:47.096+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.485 seconds
[2023-11-26T02:52:17.449+0000] {processor.py:157} INFO - Started process (PID=1065) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:52:17.451+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:52:17.453+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:52:17.452+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:52:17.855+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:52:17.854+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:52:17.856+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:52:17.868+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.425 seconds
[2023-11-26T02:52:48.228+0000] {processor.py:157} INFO - Started process (PID=1074) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:52:48.231+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:52:48.232+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:52:48.232+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:52:48.592+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:52:48.591+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:52:48.592+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:52:48.603+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.381 seconds
[2023-11-26T02:53:18.950+0000] {processor.py:157} INFO - Started process (PID=1083) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:53:18.952+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:53:18.954+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:53:18.953+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:53:19.340+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:53:19.340+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:53:19.341+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:53:19.354+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.410 seconds
[2023-11-26T02:53:49.714+0000] {processor.py:157} INFO - Started process (PID=1092) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:53:49.716+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:53:49.717+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:53:49.717+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:53:50.055+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:53:50.054+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:53:50.056+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:53:50.066+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.360 seconds
[2023-11-26T02:54:20.369+0000] {processor.py:157} INFO - Started process (PID=1101) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:54:20.371+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:54:20.372+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:54:20.372+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:54:20.750+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:54:20.749+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:54:20.751+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:54:20.762+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.399 seconds
[2023-11-26T02:54:51.158+0000] {processor.py:157} INFO - Started process (PID=1110) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:54:51.162+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:54:51.163+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:54:51.163+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:54:51.571+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:54:51.570+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:54:51.572+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:54:51.585+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.431 seconds
[2023-11-26T02:55:21.933+0000] {processor.py:157} INFO - Started process (PID=1119) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:55:21.935+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:55:21.936+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:55:21.936+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:55:22.343+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:55:22.342+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:55:22.343+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:55:22.373+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.446 seconds
[2023-11-26T02:55:52.708+0000] {processor.py:157} INFO - Started process (PID=1128) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:55:52.710+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:55:52.712+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:55:52.711+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:55:53.129+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:55:53.128+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:55:53.129+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:55:53.141+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.440 seconds
[2023-11-26T02:56:23.484+0000] {processor.py:157} INFO - Started process (PID=1137) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:56:23.486+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:56:23.487+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:56:23.486+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:56:23.864+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:56:23.863+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:56:23.864+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:56:23.875+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.395 seconds
[2023-11-26T02:56:54.277+0000] {processor.py:157} INFO - Started process (PID=1146) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:56:54.279+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:56:54.280+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:56:54.280+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:56:54.666+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:56:54.665+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:56:54.666+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:56:54.679+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.408 seconds
[2023-11-26T02:57:25.064+0000] {processor.py:157} INFO - Started process (PID=1155) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:57:25.067+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:57:25.069+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:57:25.068+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:57:25.484+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:57:25.483+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:57:25.485+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:57:25.496+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.439 seconds
[2023-11-26T02:57:55.790+0000] {processor.py:157} INFO - Started process (PID=1164) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:57:55.792+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:57:55.793+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:57:55.793+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:57:56.181+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:57:56.181+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:57:56.182+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:57:56.195+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.411 seconds
[2023-11-26T02:58:26.611+0000] {processor.py:157} INFO - Started process (PID=1173) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:58:26.613+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:58:26.614+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:58:26.614+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:58:27.054+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:58:27.052+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:58:27.055+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:58:27.068+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.463 seconds
[2023-11-26T02:58:57.424+0000] {processor.py:157} INFO - Started process (PID=1182) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:58:57.427+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:58:57.428+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:58:57.427+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:58:57.851+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:58:57.850+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:58:57.852+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:58:57.864+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.445 seconds
[2023-11-26T02:59:28.284+0000] {processor.py:157} INFO - Started process (PID=1191) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:59:28.287+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:59:28.288+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:59:28.288+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:59:28.717+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:59:28.716+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:59:28.718+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:59:28.734+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.458 seconds
[2023-11-26T02:59:59.170+0000] {processor.py:157} INFO - Started process (PID=1200) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:59:59.171+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T02:59:59.173+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:59:59.172+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:59:59.588+0000] {logging_mixin.py:154} INFO - [2023-11-26T02:59:59.587+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T02:59:59.589+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T02:59:59.605+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.439 seconds
[2023-11-26T03:00:30.026+0000] {processor.py:157} INFO - Started process (PID=1209) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:00:30.030+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T03:00:30.032+0000] {logging_mixin.py:154} INFO - [2023-11-26T03:00:30.031+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:00:30.488+0000] {logging_mixin.py:154} INFO - [2023-11-26T03:00:30.487+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T03:00:30.488+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:00:30.505+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.502 seconds
[2023-11-26T03:01:00.895+0000] {processor.py:157} INFO - Started process (PID=1218) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:01:00.896+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T03:01:00.897+0000] {logging_mixin.py:154} INFO - [2023-11-26T03:01:00.897+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:01:01.305+0000] {logging_mixin.py:154} INFO - [2023-11-26T03:01:01.304+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T03:01:01.306+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:01:01.319+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.427 seconds
[2023-11-26T03:01:31.744+0000] {processor.py:157} INFO - Started process (PID=1227) to work on /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:01:31.746+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/outcomes_dag.py for tasks to queue
[2023-11-26T03:01:31.747+0000] {logging_mixin.py:154} INFO - [2023-11-26T03:01:31.746+0000] {dagbag.py:536} INFO - Filling up the DagBag from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:01:32.190+0000] {logging_mixin.py:154} INFO - [2023-11-26T03:01:32.189+0000] {dagbag.py:346} ERROR - Failed to import: /opt/airflow/dags/outcomes_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 342, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/outcomes_dag.py", line 38, in <module>
    python_callable= fetch_data_and_save_csv,
NameError: name 'fetch_data_and_save_csv' is not defined
[2023-11-26T03:01:32.190+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/outcomes_dag.py
[2023-11-26T03:01:32.221+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/outcomes_dag.py took 0.485 seconds
